sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/implementation/instrumentation/EventHubsConsumerInstrumentation.java

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Similarly, is it possible to mock some tests for this class?

Reply:
It is tested with metrics and tracing tests, 100% coverage